plugins/out_pgsql/pgsql_connections.c

```diff
@@ -143,6 +143,10 @@ int pgsql_next_connection(struct flb_pgsql_config *ctx)
     struct mk_list *head;
     int ret_conn = 1;
 
+    if (ctx ==NULL) {
```

Comment:
another space it's missing here too

Reply:
fixed in the commit: out_pgsql: code-format &if config err then log err